uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/task-list.js

```diff
@@ -81,12 +81,12 @@ define([
             $scope.dueToday = 0;
             $scope.dueThisWeek = 0;
             $scope.overdue = 0;
-            $scope.taskList = taskList;
-            $scope.taskListFiltered = [];
-            $scope.taskListLimit = 5;
-            $scope.taskListOngoing = [];
-            $scope.taskListResolved = [];
-            $scope.taskListResolvedLoaded = false;
+            $scope.list = taskList;
```

Comment:
There is lot of common code between task-list.js and document-list.js. 
https://www.diffchecker.com/ry85ypxg
We can create a parent controller to reduce redundancy.